tuf/api/metadata.py

```diff
@@ -104,7 +104,10 @@ class Metadata(Generic[T]):
     "[Root]" is not validated at runtime (as pure annotations are not available
     then).
 
-    Attributes:
+    *All parameters named below are not just constructor arguments but also
+    instance attributes.*
+
+    Args:
```

Comment:
nit: I'm not the person who is going to insist on blindly following the guidelines but should we explain this "deviation" to potential new contributors somewhere? (The style guideline says public attributes of classes should be documented)

Reply:
It's certainly not orthodox... The explanation you quoted is the best I could come up with but I'll definitely take suggestions.

I'm against duplicating the docs so both arguments and attributes are documented as long as the arguments and attributes are the same ... if they end up diverging the situation is different.